### PR TITLE
README: Adding coverity badge

### PR DESCRIPTION
**Pull request by shamisp**

### CODE REVIEW

**Comment by shamisp** [comment]
@miked-mellanox , please take a look

**Comment by shamisp** [comment]
autobot build

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/200/ for details. (Mellanox internal link)

**Comment by miked-mellanox** [comment]
``` shell
21:49:07   CXXLD    gtest
21:49:07 make[2]: Leaving directory `/scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node/build-test/test/gtest'
21:49:07 make[2]: Entering directory `/scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node/build-test'
21:49:07 make[2]: *** No rule to make target `README', needed by `all-am'.  Stop.
21:49:07 make[2]: Leaving directory `/scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node/build-test'
21:49:07 make[1]: *** [all-recursive] Error 1
21:49:07 make[1]: Leaving directory `/scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node/build-test'
21:49:07 make: *** [all] Error 2
21:49:07 Build step 'Execute shell' marked build as failure

```

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/201/ for details. (Mellanox internal link)

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com:8888/jenkins/job/gh-ucx-pr/765/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com:8888/jenkins/job/gh-ucx-pr/768/ for details.

**Comment by shamisp** [comment]
It drives me nuts...

File: Makefile.am [hunk 1]
```diff
@@ -17,8 +17,8 @@ doxygen_doc_files = $(noinst_HEADERS)
 perftest_dir = $(pkgdatadir)/perftest
 doc_dir = $(pkgdatadir)/doc
 
-dist_perftest__DATA = contrib/ucx_perftest_config/msg_pow2 contrib/ucx_perftest_config/README contrib/ucx_perftest_config/test_types contrib/ucx_perftest_config/transports
-dist_doc__DATA = README
+dist_perftest__DATA = contrib/ucx_perftest_config/msg_pow2 contrib/ucx_perftest_config/README.md contrib/ucx_perftest_config/test_types contrib/ucx_perftest_config/transports
```
Comment (miked-mellanox) [inline Makefile.am]: README.md does not comply to gnu coding convention.
we should have README.md for github only and use README for autotools.

Comment (shamisp) [inline Makefile.am]: But I don't want to have duplicated files ? The latest commit fixes the issue that everything supposed to work

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/202/ for details. (Mellanox internal link)

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com:8888/jenkins/job/gh-ucx-pr/770/ for details.

**Comment by shamisp** [comment]
@yosefe - Jenkins is not doing well ?
```
23:52:19 Build timed out (after 60 minutes). Marking the build as failed.
23:52:19 Build was aborted
23:52:19 TAP Reports Processing: START
23:52:19 Looking for TAP results report in workspace using pattern: **/*.tap
23:52:19 Did not find any matching files.
```

**Comment by miked-mellanox** [comment]
job runtime exceeded 60m, increased to 90

**Comment by miked-mellanox** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com:8888/jenkins/job/gh-ucx-pr/776/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com:8888/jenkins/job/gh-ucx-pr/777/ for details.

**Comment by miked-mellanox** [comment]
:+1:

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/207/ for details. (Mellanox internal link)

**Comment by shamisp** [comment]
@miked-mellanox check MellanoxLab errors and we can ignore it (some script error).

**Pull request merged**